src/main/java/digital/slovensko/autogram/ui/gui/SigningDialogController.java

```diff
@@ -52,6 +52,10 @@ public SigningDialogController(SigningJob signingJob, Autogram autogram, GUI gui
 
     public void initialize() {
         refreshSigningKey();
+
+        mainBox.setPrefWidth(signingJob.getPreviewWidth());
+        mainBox.setMinWidth(signingJob.getPreviewWidth());
```

Comment:
Tu by som bol asi opatrnejsi, ak si niekto vypyta xxl a nemas taky monitor tak sa stane to, ze to bude prilis siroke? Ci nechame toto na integratora?

Reply:
nechal by som na skôr integrátora. Akože, možno vieme nestaviť tiež nejakú maxWidth